### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by robswc** [robswc/nadocast-ui#14] [General]
Please leave a post here if you're interested in contributing!  In time I will provide more structure and make contributing easier but for now just getting a chat/discussion going will be good enough!

**Comment by duynlq** [comment]
Hi there, I am interested in contributing because I am a Texas State alumni! I worked with Python and data tools in grad school and a little at work, so I have some experience in web app development as well as data manipulation and predictive analysis. Excited to work with you! 

Using Dash-plotly is really nice, it's like building JS elements using Python. Do you have an idea of what to work on next yet, like a task list?

**Reply by robswc** [reply]
Hey! That's awesome!

I do have some ideas!  I'm just sorta throwing everything here to keep it simple: https://github.com/robswc/nadocast-ui/issues

Everything marked as "enhancement" would be something I'm looking to add but haven't yet.  In the spirit of open source feel free to add anything you feel will improve the project!
